### ISSUE TITLE
WordPress Setup and adding Ray

### ISSUE THREAD

**Issue by palmiak** [patchstack/academy#12]
There is an alternative to xdebug called [Ray](https://myray.app/docs/getting-started/introduction) - as I'm not a researcher I'm not sure if this solution will also be good enough. It's for sure great for development.

@dhakalananda @rafiem

**Comment by dhakalananda** [comment]
@palmiak 

I believe we should just stick to XDebug because it is the most widely used in the security community. If we introduce a completely new tool, it might take time for researchers to learn and get familiar with, and have nothing in return for that time as XDebug will do all the required stuffs.

Looks like Ray offers a paid option as well which we don't want since we already have a fully open-source option available.

What do you say @rafiem ?

**Comment by palmiak** [comment]
It's more to mention this as another way rather than replacing xdebug. For sure setting up Ray is simpler. The biggest downside is the fact it's a paid product (the trail is short).

**Comment by rafiem** [comment]
Yes, let's stick with xdebug for now and we can explore alternatives such as Ray, etc.
